### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by podcastinator**
This PR updates the landing page of the repo, and pulls in doc updates from the hackathon branch.

### CODE REVIEW

File: doc/build.rst
Comment (chester-leung) [inline doc/build.rst]: Could you delete the extra space here? There seem to be two spaces after the `-`

File: doc/build.rst
Comment (chester-leung) [inline doc/build.rst]: Maybe we can remove "and check out the required branch." I think this is a holdover from the hackathon branch?

File: doc/build.rst
Comment (chester-leung) [inline doc/build.rst:113]: This is no longer relevant, right? #19 will automatically create the .conf file from the build. 

Do you think we should make the corresponding changes to the documentation in this PR, or wait until #19 is merged and then create another PR with updated documentation?

File: doc/get_started.rst
Comment (chester-leung) [inline doc/get_started.rst:20]: This may also need to be updated once #19 is merged in.

**Review by chester-leung — CHANGES_REQUESTED** [review]

File: doc/build.rst
Comment (podcastinator) [reply doc/build.rst]: Fixed!

File: doc/build.rst
Comment (podcastinator) [reply doc/build.rst]: Good catch, fixed!

File: doc/build.rst
Comment (podcastinator) [reply doc/build.rst:113]: I think we should fix this in #19. So let that PR be for now, will update it once you merge this one in.

File: doc/get_started.rst
Comment (podcastinator) [reply doc/get_started.rst:20]: Yes. Perhaps #19 should remove enclave flags from the demos since the user won't need to configure them anymore at runtime.

**Comment by podcastinator** [comment]
Addressed the nits. Will update #19 to address the changes required by it.

**Comment by chester-leung** [comment]
Looks good. Thanks!

**Review by chester-leung — APPROVED** [review]

**Pull request merged**
